ios/Classes/CountlyFlutterPlugin.m

```diff
@@ -1538,6 +1538,21 @@ - (void)populateConfig:(NSDictionary *)_config {
         if (startTSOverride) {
             [config.apm setAppStartTimestampOverride:[startTSOverride longLongValue]];
         }
+
+        // Internal Limits ---------------------
+        NSNumber *maxKeyLength = _config[@"maxKeyLength"];
+        if (maxKeyLength) {
+            config.maxKeyLength = [maxKeyLength intValue];
+        }
+        NSNumber *maxValueSize = _config[@"maxValueSize"];
+        if (maxValueSize) {
+            config.maxValueLength = [maxValueSize intValue];
+        }
+        NSNumber *maxSegmentationValues = _config[@"maxSegmentationValues"];
+        if (maxSegmentationValues) {
+            config.maxSegmentationValues = [maxSegmentationValues intValue];
+        }
```

Comment:
maybe you can add other remaning ones so it will be added in the future as a note